### PR TITLE
Add og tags for facebook

### PR DESCRIPTION
**Pull request by jhack32**
https://blog.kissmetrics.com/open-graph-meta-tags/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -24,6 +24,15 @@
 
   </script>
   <link type="text/plain" rel="author" href="humans.txt" />
+  
+  <meta property="og:locale" content="en_US" />
+  <meta property="og:type" content="website" />
+  <meta property="og:title" content="Cortlandt Labs" />
+  <meta property="og:description" content="Cortlandt Labs is a technology consulting agency started in NYC."
```
Comment (dijonkitchen) [inline index.html]: Missing self-closing tag.

File: index.html [hunk 2]
```diff
@@ -24,6 +24,15 @@
...
+  <meta property="og:site_name" content="Cortlandt Labs"/>
+  <meta property="og:url" content="http://cortlandtlabs.co/"/>
+  <meta property="og:image" content="https://raw.githubusercontent.com/cortlandtlabs/cortlandtlabs.github.io/master/assets/Cortlandt-Labs-logo.png"/>
```
Comment (dijonkitchen) [inline index.html]: Can we use a relative link here?

File: index.html [hunk 3]
```diff
@@ -24,6 +24,15 @@
 
   </script>
   <link type="text/plain" rel="author" href="humans.txt" />
+  
+  <meta property="og:locale" content="en_US" />
```
Comment (dijonkitchen) [inline index.html]: Perhaps group all these meta tags on the top with the other meta tags and add a line comment for Facebook Open Graph?

**Review by dijonkitchen — CHANGES_REQUESTED** [review]
Cool stuff! See my comments. Did you also check it on Facebook's crawler to test it? https://developers.facebook.com/docs/sharing/webmasters

Do we also want to add Twitter stuff while we're at it?

File: index.html (hunk 2 above)
Comment (jhack32) [reply index.html]: I don't think so

**Review by jhack32 — COMMENTED** [review]
Changes were made based on comments. I don't think we need to add twitter tags because if twitter tags can't be located, it will look at the og tags.

**Review by dijonkitchen — APPROVED** [review]

File: index.html (hunk 2 above)
Comment (dijonkitchen) [reply index.html]: I'll look into it later, but the absolute link should work fine for now.

**Comment by dijonkitchen** [comment]
👍 Love the new review system!

**Review by aattsai — APPROVED** [review]

**Pull request merged**